test/ui/LoginTest.js

```diff
@@ -0,0 +1,122 @@
+var webdriver = require('selenium-webdriver');
+var chai = require("chai");
+var expect = chai.expect;
+
+describe('Login Page', function() {
+    this.timeout(50000); // Set timeout to 5 seconds
+    var driver;
+    var page;
+
+    it('should have the correct title', async function(){
+        driver = new webdriver.Builder().forBrowser('chrome').build();
+        page = await driver.get('http://localhost:3000/login'); 
+        const title = await driver.getTitle();
+        expect(title).to.equal('Login Or Sign Up'); 
+    });
+
+    it('should redirect admin to admin menu page with valid credentials', async function() {
+        page = await driver.get('http://localhost:3000/login'); 
+        const usernameInput = await driver.findElement(webdriver.By.id('username')); 
+        const passwordInput = await driver.findElement(webdriver.By.id('password')); 
+        const loginButton = await driver.findElement(webdriver.By.id('login')); 
+
+        await usernameInput.sendKeys('JohnsSuperAdmin1');
+        await passwordInput.sendKeys('Kainos&Ulster23');
```

Comment:
Should not be using valid usernames and passwords in your tests. If you require valid creds, you will need to store these in environment variables.